rbac/internal/views.py

```diff
@@ -50,16 +50,22 @@ def destructive_ok():
     return now < settings.INTERNAL_DESTRUCTIVE_API_OK_UNTIL
 
 
-def tenant_is_modified():
+def tenant_is_modified(schema_name):
     """Determine whether or not the tenant is modified."""
-    return (Role.objects.filter(system=True).count() != Role.objects.count()) or (
+    has_custom_data = (Role.objects.filter(system=True).count() != Role.objects.count()) or (
```

Comment:
LGTM overall, wonder if we could put schema_exists(schema_name) before this line, so if it is false, we don't have to do these queries.

Reply:
We can't put it before this line in this method, because the presence of a schema alone isn't enough to tell us if the tenant has been modified. I'd initially considered placing it in `tenant_is_unmodified` as:
```
def tenant_is_unmodified(schema_name):
    """Determine whether or not the tenant is unmodified."""
    if not schema_exists(schema_name):
        return True
    return not tenant_is_modified(schema_name)
```
But that would have only fixed the `tenant_is_unmodified` method, and seemed a bit more confusing. What are your thoughts?

Reply:
Pushed an update in 7aed3a4